src/server/templates/base.html

```diff
@@ -78,16 +96,54 @@ <h1 class="col-3 flex-grow-1">
 
     <div class="container fill">
       <div class="row">
-        <div class="col">
+        <div class="col-12">
           <main role="main" class="">
             {% block content %} {% endblock %}
+            <br />
```

Comment:
If this is for footer margin,  can't we rather replace it by adding:

```
<footer class='mt-5' ...
```

below?